app/controllers/admin/loans_controller.rb

```diff
@@ -39,6 +39,7 @@ def show
     @steps = @loan.project_steps
     @calendar_events_url = "/admin/calendar_events?project_id=#{@loan.id}"
     @active_tab = params[:tab].presence || "details"
+    @questionnaire_filter = params[:filter] || "criteria"
```

Comment:
There is duplication in the show and questionnaires methods in this controller. Is there a recommended way to adjust this?